samples/config.default.js

```diff
@@ -1,36 +1,57 @@
-// User-specific configuration
-exports.voice = 'en-US_MichaelVoice';
-exports.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
+// User-specific CONFIGURATION
+exports.config = {}
+exports.config.voice = {"gender": "female", "language": "en-US"};
+exports.config.conversationWorkspaceId = ''; // replace with the workspace identifier of your conversation
 
-// Create the credentials object for export
-exports.credentials = {};
+// Card and Device found in command arecord -l of your hardware. In general 0,0 or 1,0 or 0,1
+// If bot is closing without wait for listening you maybe have to change that parameter
+exports.config.microphone = "plughw:0,0";
+// Attention word that bot are waiting for
+exports.config.attentionWord = 'TJ';
+// To see log messages
+exports.config.verboseLogging = false;
 
-// Watson Conversation
-// https://www.ibm.com/watson/developercloud/conversation.html
-exports.credentials.conversation = {
-    password: '',
-    username: ''
+exports.config.ledPin = 8;
+exports.config.servoPin = 7;
+exports.config.ttsReconnect = true;
+// Confidence threshold for tags from visual recognition service. Tags below this will be ignored.
+exports.config.visionConfidenceThreshold = 0.5;
+exports.config.visionTextConfidenceThreshold = 0.1;
+exports.config.cameraParams = {
+    height: 720,
+    width: 960
+}
+
+// Create the CREDENTIALS object for export
+exports.credentials = {
 };
 
 // Watson Speech to Text
 // https://www.ibm.com/watson/developercloud/speech-to-text.html
 exports.credentials.speech_to_text = {
-    password: '',
-    username: ''
+  "username": "",
```

Comment:
is it necessary to put these in quotes?

also, i think we wanted to list password first because that was the order that bluemix gives the service credentials (it makes copy/paste easier)